src/ci/docker/README.md

```diff
@@ -257,6 +257,21 @@ For targets: `i586-unknown-linux-gnu`
 (\*) Compressed debug is enabled by default for gas (assembly) on Linux/x86 targets,
      but that makes our `compiler_builtins` incompatible with binutils < 2.32.
 
+### `loongarch64-linux-gnu.defconfig`
+
+For targets: `loongarch64-unknown-linux-gnu`
+
+- Path and misc options > Prefix directory = /x-tools/${CT\_TARGET}
+- Path and misc options > Patches origin = Per global policy
+- Target options > Target Architecture = loongarch
+- Target options > Bitness = 64-bit
+- Operating System > Target OS = linux
+- Operating System > Linux kernel version = 5.19.16
+- Binary utilities > Version of binutils = 2.40
+- C-library > glibc version = 2.36
+- C compiler > gcc version = 12.2.0
```

Comment:
Are we able to raise this to GCC 13.1.0 now that it's released?

Reply:
Certainly, although GCC 13 has been released, the upgrade for crosstool-ng is still in progress. We would like to catch up with the Rust 1.71 release, and since LoongArch CI is not dependent on GCC 13, I propose that we upgrade after crosstool-ng is ready, possibly through a new PR.

Reply:
If crosstool-ng is ready earlier than expected, I would be happy to include it in this PR.

Reply:
Ah yes. I forgot we privately talked about this a while ago.